scipy/spatial/kdtree.py

```diff
@@ -199,6 +199,10 @@ class KDTree(object):
             >>> import sys
             >>> sys.setrecursionlimit(10000)
 
+    See Also
+    --------
+    cKDTree : Implementation of `KDTree` in Cython
```

Comment:
Okay, if we're going to quibble about this, then let me point out: yes, the <b>functionality</b> is written in C++, but `kdtree.py` is the interface and functionality all packaged into one class.  Thus, to say that "KDTree implemented in C++" is not entirely accurate because it does not account for the interface portion.